### PR TITLE
Update text_classification_rnn.ipynb

### PR DESCRIPTION
**Pull request by sushreebarsa**
Changed the threshold value from 0.0 to 0.5 
Fixes #53316

### CODE REVIEW

**Comment by foxik** [comment]
Not a TF developer, but this PR is incorrect -- the condition `prediction is >= 0.0` is actually fine.

The reason is that the model is predicting logits, not probabilities, and logits have a threshold at 0 (while probabilities at 0.5). To see that the logits are used:
1) the model description
   ```python
   model = tf.keras.Sequential([
    encoder,
    tf.keras.layers.Embedding(
        input_dim=len(encoder.get_vocabulary()),
        output_dim=64,
        # Use masking to handle the variable sequence lengths
        mask_zero=True),
    tf.keras.layers.Bidirectional(tf.keras.layers.LSTM(64)),
    tf.keras.layers.Dense(64, activation='relu'),
    tf.keras.layers.Dense(1)
   ])
   ```
   does not use any activation function (it wouls use `sigmoid` if probabilities would be produced)
2) the loss
   ```python
   model.compile(loss=tf.keras.losses.BinaryCrossentropy(from_logits=True),
              optimizer=tf.keras.optimizers.Adam(1e-4),
              metrics=['accuracy'])
   ```
   specifies it is `from_logits`.

Cheers.

**Comment by anirudh161** [comment]
Agreed. This PR is incorrect. Closing it.
